cmake/configs/default.cmake

```diff
@@ -62,7 +62,7 @@ hunter_config(Libcxxabi VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
-hunter_config(OpenCV VERSION 3.1.0-p1)
+hunter_config(OpenCV VERSION 3.1.0-p2 CMAKE_ARGS WITH_QT=YES)
```

Comment:
> Your call

Now it will break CI testing/uploading of OpenCV. I'm working on Qt uploading to cache from my local machine so my suggestion is to pause this merge until I've finished so we can have this cool automation working. Also I don't want to make a decision myself since I can't be a guy who is responsible for every package in system. I think I should delegate end verdict to the [OpenCV team](https://github.com/orgs/hunter-packages/teams/opencv): @headupinclouds what do you think?